app/src/main/java/com/example/playlistmaker2/OtherClass.kt

```diff
@@ -21,4 +27,27 @@ public class NetworkUtils {
             }
         }
     }
-}
\ No newline at end of file
+}
+
+
+class MusicRequest(val text: String)
+
+
+    data class TrackITunes(val trackName: String,
+                       val artistName: String,
+                       val trackTimeMillis: Int,
+                       val artworkUrl100: String)
+
+
+data class SearchResponseMusic(
+    val resultCount: Int,
+    val results: List<TrackITunes>
+)
+
+interface MusicApi {
+    @GET("/search")
```

Comment:
Сюда лучше добавить query параметр entity, чтобы искать только по трекам:
```suggestion
    @GET("/search?entity=song")
```

Reply:
Очень рекомендую все таки добавить этот query параметр, иначе в поиск попадают не только треки